activiti-cloud-service-common/activiti-cloud-services-common-security-keycloak/src/main/java/org/activiti/cloud/services/common/security/config/CommonSecurityAutoConfiguration.java

```diff
@@ -0,0 +1,201 @@
+    public JwtAccessTokenPrincipalGroupsProvider keycloakAccessTokenPrincipalGroupsProvider(JwtAccessTokenProvider jwtAccessTokenProvider,
+        JwtAccessTokenValidator jwtAccessTokenValidator) {
+        return new JwtAccessTokenPrincipalGroupsProvider(jwtAccessTokenProvider,
+            jwtAccessTokenValidator);
+    }
+
+    @Bean
+    @Order(Ordered.HIGHEST_PRECEDENCE)
+    @ConditionalOnMissingBean
+    public JtwAccessTokenPrincipalRolesProvider keycloakAccessTokenPrincipalRolesProvider(JwtAccessTokenProvider keycloakAccessTokenProvider,
+                                                                                               JwtAccessTokenValidator jwtAccessTokenValidator) {
+        return new JtwAccessTokenPrincipalRolesProvider(keycloakAccessTokenProvider,
+            jwtAccessTokenValidator);
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public KeycloakPrincipalGroupsProviderChain principalGroupsProviderChain(List<PrincipalGroupsProvider> principalGroupsProviders) {
+        return new KeycloakPrincipalGroupsProviderChain(principalGroupsProviders);
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public KeycloakPrincipalRolesProviderChain principalRolesProviderChain(List<PrincipalRolesProvider> principalRolesProviders) {
+        return new KeycloakPrincipalRolesProviderChain(principalRolesProviders);
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public SecurityManager securityManager(SecurityContextPrincipalProvider authenticatedPrincipalProvider,
+                                           PrincipalIdentityProvider principalIdentityProvider,
+                                           KeycloakPrincipalGroupsProviderChain principalGroupsProvider,
+                                           KeycloakPrincipalRolesProviderChain principalRolesProviderChain) {
+        return new SecurityManagerImpl(authenticatedPrincipalProvider,
+                                               principalIdentityProvider,
+                                               principalGroupsProvider,
+                                               principalRolesProviderChain);
+    }
+
+    @Bean
+    @ConditionalOnMissingBean
+    public SecurityContextTokenProvider securityContextTokenProvider() {
+        return new JwtSecurityContextTokenProvider();
+    }
+
+    /**
+     * Defines the session authentication strategy.
+     */
+    @Bean
+    @ConditionalOnMissingBean
+    protected SessionAuthenticationStrategy sessionAuthenticationStrategy() {
+        return new RegisterSessionAuthenticationStrategy(new SessionRegistryImpl());
+    }
+
+    @Override
+    protected void configure(HttpSecurity http) throws Exception {
+        authorizationConfigurer.configure(http);
+        http
+            .authorizeRequests().anyRequest().permitAll()
+            .and()
+            .cors()
+            .configurationSource(request -> {
+                CorsConfiguration corsConfiguration = new CorsConfiguration();
```

Comment:
I forgot that we should review this configuration in order to be sure that we accept calls only from valid sources